tests/integration/test_transformers.py

```diff
@@ -67,7 +67,9 @@ def _validate_helper(validator_function, args, steps):
 
 def _is_valid_transformer(transformer_name):
     """Determine if transformer should be tested or not."""
-    invalid_names = ['IdentityTransformer', 'Dummy', 'OrderedLabelEncoder', 'CustomLabelEncoder']
+    invalid_names = [
```

Comment:
I don't know if we want this. I think if yes, we need an appropriate dataset with ID columns.

Reply:
How does the `AnonymizedFaker` get tested? Wouldn't that have similar data?

Reply:
It does not here because it has `INPUT_SDTYPE='pii'`. Following the discussion in the `EngMeeting` to not support `id sdtype` on `RDT`, should I set `INPUT_SDTYPE='pii'` to the IDGenerator rather than `INPUT_SDTYPE='id'` (as mentioned in the issue) @amontanez24?